source/tomopy/prep/normalize.py

```diff
@@ -231,9 +243,12 @@ def normalize_nf(tomo, flats, dark, flat_loc,
     flats : ndarray
         3D flat field data.
     dark : ndarray
-        3D dark field data.
+        2D or 3D dark field data.
     flat_loc : list of int
         Indices of flat field data within tomography
+    averaging : str, optional
+        'mean' or 'median', based on how the flat and dark arrays
```

Comment:
```suggestion
        'mean' or 'median', how the flat and dark arrays
```

Reply:
This documentation claims that the 'averaging' parameter controls how *both* the dark and flats are combined, but it actually only controls how the darks are combined. Please reread the documentation and to ensure that it is consistent with what is implemented.